gcloud_bigtable/row.py

```diff
@@ -416,12 +419,11 @@ def _commit_mutate(self, timeout_seconds=None):
             mutations=mutations_list,
         )
         timeout_seconds = timeout_seconds or self.timeout_seconds
-        response = self.client.data_stub.MutateRow.async(request_pb,
-                                                         timeout_seconds)
         # We expect a `._generated.empty_pb2.Empty`.
-        response.result()
+        response = self.client.data_stub.MutateRow.async(request_pb, timeout_seconds).result() if async \
```

Comment:
Instead of slash continuing the line please do

```python
        if async:
            response = self.client.data_stub.MutateRow.async(request_pb, timeout_seconds)
            response.result()
        else:
            self.client.data_stub.MutateRow(request_pb, timeout_seconds)